css/src/components/message.scss

```diff
@@ -0,0 +1,50 @@
+.message {
+	display: grid;
+	grid-template:
+		'. message-time'
+		'message-persona message-content';
+	grid-template-columns: max-content 1fr;
+	gap: $layout-1;
+
+	.message-time {
+		grid-area: message-time;
+		font-size: $font-size-9;
+	}
+
+	.message-persona {
+		grid-area: message-persona;
+		margin-block-start: $spacer-3;
+	}
+
+	.message-content {
```

Comment:
Can you add a default `font-size` to `message-content`? Or to the `message` level so there are no surprises on how it's gonna show up in different projects.